cluster/gce/configure-vm.sh

```diff
@@ -260,6 +260,13 @@ function create-salt-auth() {
   kubelet_auth_file="/srv/salt-overlay/salt/kubelet/kubernetes_auth"
   (umask 077;
     echo "{\"BearerToken\": \"${KUBELET_TOKEN}\", \"Insecure\": true }" > "${kubelet_auth_file}")
+
+  # Generate tokens for other "service accounts".  Append to known_tokens.
+  for a in scheduler controller_manager logging monitoring dns
+  do
+    token=$(dd if=/dev/urandom bs=128 count=1 2>/dev/null | base64 | tr -d "=+/" | dd bs=32 count=1 2>/dev/null)
+    echo "${token},kubernetes_builtin:$a,kubernetes_builtin:$a" >> "${known_tokens_file}"
```

Comment:
@smarterclayton ISTR you suggesting a format for namespacing usernames but I can't find the issue now.  Is there already a standard used in openshift?  Here I used `kubernetes_builtin:$USER`, but I am open to whatever.

@liggitt